src/compiler/php_plugin.cc

```diff
@@ -40,12 +41,25 @@ class PHPGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
       return true;
     }
 
+    std::vector<std::pair<grpc::string, grpc::string> > options;
+    ParseGeneratorParameter(parameter, &options);
+
+    grpc::string class_suffix;
+    for (size_t i = 0; i < options.size(); ++i) {
+      if (options[i].first == "class_suffix") {
```

Comment:
The PR #11773 handling the `parameter` has *just* been merged to `master` yesterday. So no one is using it yet. I want to make changes on top of that to make sure it's more flexible when this does get released.